src/components/Calendar.tsx

```diff
@@ -29,13 +29,20 @@ const { DOT_SIZE, MAX_DOTS_PER_DAY } = CALENDAR;
 type Props = {
   isListRefreshing: boolean;
   query: string;
-  queryVariables?: { dateRange?: string[]; contentContainerId?: number };
+  queryVariables: { dateRange?: string[]; contentContainerId?: number };
   navigation: StackNavigationProp<any>;
+  additionalData?: any;
 };
 
 const today = moment().format('YYYY-MM-DD');
 
-export const Calendar = ({ isListRefreshing, query, queryVariables, navigation }: Props) => {
+export const Calendar = ({
+  isListRefreshing,
+  query,
+  queryVariables,
+  navigation,
+  additionalData
+}: Props) => {
```

Comment:
Wouldn't alphabetical sorting be better?

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/940/commits/f653378eefcca79cef1def80dfa038e995b1bccf